CONTRIBUTING.md

```diff
@@ -45,6 +45,66 @@ Before you can start contributing to the CyberArk Secrets Provider for Kubernete
     
     b. For detailed setup instructions for OSS, see [CyberArk Secrets Provider for Kubernetes for OSS](https://docs.conjur.org/Latest/en/Content/Integrations/Kubernetes_deployApplicationsConjur-k8s-Secrets.htm).
     
+### Deploy a Local Dev Environment (K8s)
+
+You can now deploy a local development environment for Kubernetes using [Docker Desktop](https://www.docker.com/products/docker-desktop). Docker Desktop provides a convenient way to deploy and develop from your machine against a locally deployed cluster. 
+
+#### Prerequisites
+
+1. [Docker Desktop](https://www.docker.com/products/docker-desktop) installed
+
+1. Kubernetes enabled in Docker Desktop
+
+    1. Navigate to Docker Preferences
+    
+    1. Click on the Kubernetes tab and "Enable Kubernetes"
+    
+1. The Secrets Provider for K8s uses the [Kubernetes Conjur deploy](https://github.com/cyberark/kubernetes-conjur-deploy/blob/master/CONTRIBUTING.md) repository to deploy Conjur/DAP on Kubernetes. 
+   By default, 2.0 Gib of memory is allocated to Docker on your computer.
+   
+   To successfully deploy a DAP cluster (Master + Followers + Standbys), you will need to increase the memory limit to 6 Gib. To do so, perform the following:
+   
+   1. Navigate to Docker preferences
+   
+   1. Click on "Advanced" and slide the "Memory" bar to 6
+
+#### Deploy
+
+To deploy a local development environment, perform the following:
+
+1. Run `./bin/build` to build the Secrets Provider image locally 
+
+1. Ensure you are in the proper local context otherwise, the deployment will not run successfully
+
+    Run ` kubectl config current-context` to verify which context you are currently in so if needed, you can switch back to it easily
+
+    Run `kubectl config use-context docker-desktop` to switch to a local context. This is the context you will need to run the development environment
+
+1. Navigate to `bootstrap.env` and uncomment the `Local DEV Env` section, ensuring that `DEV=true`
+
+1. Run `./bin/start --dev --gke`, appending `--oss` or `--dap` according to the environment that needs to be deployed
+
+1. To view the pod(s) that were deployed and the Secrets Provider logs, run `kubectl get pods` and `kubectl logs <pod-name> -c cyberark-secrets-provider` respectively
```

Comment:
how about adding also the pods of conjur? adding `-n local-conjur`? (is it `local-conjur`?)